packages/core/src/delegation.js

```diff
@@ -162,16 +162,46 @@ export class Delegation {
     this.root = root
     this.blocks = blocks
 
-    /** @type {API.AttachedLinkSet} */
-    this.attachedLinks = new Set()
-
     Object.defineProperties(this, {
       blocks: {
         enumerable: false,
       },
     })
   }
 
+  /**
+   * @returns {API.AttachedLinkSet}
+   */
+  get attachedLinks() {
+    const _attachedLinks = new Set()
+    const ucanView = decode(this.root)
```

Comment:
```suggestion
    const ucanView = this.data
```

we already have a getter for this